services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java

```diff
@@ -0,0 +1,416 @@
+   */
+  public static final String INCIDENT_ROAD_HAZARD = "road_hazard";
+
+  /**
+   * {@link IncidentType} weather.
+   */
+  public static final String INCIDENT_WEATHER = "weather";
+
+  /**
+   * Incident type.
+   */
+  @Retention(RetentionPolicy.SOURCE)
+  @StringDef({
+    INCIDENT_ACCIDENT,
+    INCIDENT_CONGESTION,
+    INCIDENT_CONSTRUCTION,
+    INCIDENT_DISABLED_VEHICLE,
+    INCIDENT_LANE_RESTRICTION,
+    INCIDENT_MASS_TRANSIT,
+    INCIDENT_MISCELLANEOUS,
+    INCIDENT_OTHER_NEWS,
+    INCIDENT_PLANNED_EVENT,
+    INCIDENT_ROAD_CLOSURE,
+    INCIDENT_ROAD_HAZARD,
+    INCIDENT_WEATHER
+  })
+  public @interface IncidentType {
+  }
+
+  /**
+   * {@link ImpactType} unknown.
+   */
+  public static final String IMPACT_UNKNOWN = "unknown";
+
+  /**
+   * {@link ImpactType} critical.
+   */
+  public static final String IMPACT_CRITICAL = "critical";
+
+  /**
+   * {@link ImpactType} major.
+   */
+  public static final String IMPACT_MAJOR = "major";
+
+  /**
+   * {@link ImpactType} minor.
+   */
+  public static final String IMPACT_MINOR = "minor";
+
+  /**
+   * {@link ImpactType} low.
+   */
+  public static final String IMPACT_LOW = "low";
+
+  /**
+   * Impact type.
+   */
+  @Retention(RetentionPolicy.SOURCE)
+  @StringDef({
+    IMPACT_UNKNOWN,
+    IMPACT_CRITICAL,
+    IMPACT_MAJOR,
+    IMPACT_MINOR,
+    IMPACT_LOW
+  })
+  public @interface ImpactType {
+  }
+
+  /**
+   * Unique identifier for incident. It might be the only one <b>non-null</b> filed which meant
+   * that incident started on previous leg and one has an incident with the same <b>id</b>.
+   */
+  @NonNull
+  public abstract String id();
+
+  /**
+   * One of incident types.
+   *
+   * @see IncidentType
+   */
+  @Nullable
+  @IncidentType
+  public abstract String type();
+
+  /**
+   * <b>True</b> if road is closed and no possibility to pass through there. <b>False</b>
+   * otherwise.
+   */
+  @Nullable
+  public abstract Boolean closed();
+
+  /**
+   * Quantitative descriptor of congestion.
+   */
+  @Nullable
+  public abstract Congestion congestion();
+
+  /**
+   * Human-readable description of the incident suitable for displaying to the users.
+   */
+  @Nullable
+  public abstract String description();
+
+  /**
+   * Human-readable long description of the incident suitable for displaying to the users.
+   */
+  @Nullable
+  @SerializedName("long_description")
+  public abstract String longDescription();
+
+  /**
+   * Severity level of incident.
+   *
+   * @see ImpactType
+   */
+  @Nullable
+  @ImpactType
+  public abstract String impact();
+
+  /**
+   * Sub-type of the incident.
+   */
+  @Nullable
+  @SerializedName("sub_type")
+  public abstract String subType();
+
+  /**
+   * Sub-type-specific description.
+   */
+  @Nullable
+  @SerializedName("sub_type_description")
+  public abstract String subTypeDescription();
+
+  /**
+   * AlertC codes.
+   *
+   * @see <a href="https://www.iso.org/standard/59231.html">AlertC</a>
```

Comment:
NIT Should we use `{@link}` instead?

Reply:
a http(s) link is not working with `{@link}`